<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tapahtumat</class>
 <widget class="QDialog" name="Tapahtumat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelNimi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nimi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tunnus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTunnus">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LÃ¤hiosoite:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOsoite">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Puhelinnumero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPuhelin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSulje">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
